### PR TITLE
3 Fixes

### PR DESCRIPTION
**Pull request by dajinchu**
1. Turn off the cf-connecting warning
2. Updater use v9
3. Only show classes that have sections

Do not merge unless scrapers have run recently. Don't want the updater to spam people when we merge!

### CODE REVIEW

File: backend/updater.js [hunk 1]
```diff
@@ -21,7 +19,7 @@ class Updater {
   constructor() {
     // 5 min if prod, 30 sec if dev.
```
Comment (NEUDitao) [inline backend/updater.js]: should update the comment to reflect the time change, or move the time intervals into `const`s

File: backend/updater.js [hunk 2]
```diff
@@ -204,6 +177,7 @@ class Updater {
       // Count how many sections are present in the new but not in the old.
       let count = 0;
       if (aNewClass.sections) {
+        macros.log(oldClass);
```
Comment (NEUDitao) [inline backend/updater.js]: add something before `oldClass` indicating reason for log?

File: backend/elastic.js [hunk 3]
```diff
@@ -273,7 +273,14 @@ class Elastic {
             filter: {
               bool: {
                 should: [
-                  { term: { 'class.termId': termId } },
+                  {
+                    bool:{
```
Comment (NEUDitao) [inline backend/elastic.js:277]: I'm not an ES expert, but why is it `bool:{should:[{bool:{must:[`  ?? if this is something that ES needs to work, it's fine

**Review by NEUDitao — APPROVED** [review]
lgtm! just nitpicks. Not an es expert for the elastic stuff, but all the changes look solid

File: backend/updater.js (hunk 1 above)
Comment (dajinchu) [reply backend/updater.js]: good catch. didn't mean to change it

File: backend/updater.js (hunk 2 above)
Comment (dajinchu) [reply backend/updater.js]: oop. removing it

File: backend/elastic.js (hunk 3 above)
Comment (dajinchu) [reply backend/elastic.js:277]: yeah its an "A OR (B AND C)" query

**Pull request merged**